test/functional/feature_fork_choice_parallel_justifications.py

```diff
@@ -110,124 +110,123 @@ def generate_epoch_and_vote(node, finalizer, finalizer_address, prevtx):
-                                        'currentEpoch': 8,
-                                        'lastJustifiedEpoch': 7,
-                                        'lastFinalizedEpoch': 3,
+        assert_finalizationstate(node, {'currentDynasty': 2,
+                                        'currentEpoch': 6,
+                                        'lastJustifiedEpoch': 5,
+                                        'lastFinalizedEpoch': 2,
                                         'validators': 1})
 
         self.log.info('node successfully switched to the longest justified fork')
 
         # create longer justified epoch on the previous fork1
         # node must switch ("zag") to this fork
-        #                                J              J
-        #                             - e6 - e7 - e8 - e9 - e10 fork1 node
-        # F    F    F    F    J       |
-        # e0 - e1 - e2 - e3 - e4 - e5 -
-        #                             |       J
-        #                             - e6 - e7 - e8 fork2
+        #                     J              J
+        #                   - e4 - e5 - e6 - e7 - e8 fork1 node
+        # F    F    F       |
+        # e0 - e1 - e2 - e3 -
+        #                   |      J
+        #                   - e4 - e5 - e6 fork2
         generate_block(fork1, count=5)
         sync_node_to_fork(finalizer, fork1)
         vtx1 = generate_epoch_and_vote(fork1, finalizer, finalizer_address, vtx1)
-        assert_equal(fork1.getblockcount(), 50)
-        assert_finalizationstate(fork1, {'currentDynasty': 3,
-                                         'currentEpoch': 10,
-                                         'lastJustifiedEpoch': 9,
-                                         'lastFinalizedEpoch': 3,
+        assert_equal(fork1.getblockcount(), 40)
+        assert_finalizationstate(fork1, {'currentDynasty': 2,
+                                         'currentEpoch': 8,
+                                         'lastJustifiedEpoch': 7,
+                                         'lastFinalizedEpoch': 2,
                                          'validators': 1})
 
         assert_not_equal(fork1.getbestblockhash(), fork2.getbestblockhash())
         sync_node_to_fork(node, fork1)
-        assert_finalizationstate(node, {'currentDynasty': 3,
-                                        'currentEpoch': 10,
-                                        'lastJustifiedEpoch': 9,
-                                        'lastFinalizedEpoch': 3,
+        assert_finalizationstate(node, {'currentDynasty': 2,
+                                        'currentEpoch': 8,
+                                        'lastJustifiedEpoch': 7,
+                                        'lastFinalizedEpoch': 2,
                                         'validators': 1})
 
         self.log.info('node successfully switched back to the longest justified fork')
 
         # UNIT-E TODO: node must follow longest finalized chain
```

Comment:
it's a separate thing, I'd like to minimize changes which are not part of this PR

Reply:
I suggest removing the whole UNIT-E TODO section alongside the following code until the end of the function.